### PR TITLE
Enable the load command and use that for loading the uEnv and bootscript

### PR DESCRIPTION
**Pull request by shadeslayer**

### CODE REVIEW

File: include/configs/sun50iw1p1.h [hunk 1]
```diff
@@ -352,23 +352,23 @@
 	"initrd_filename=initrd.img\0" \
 	"bootenv_filename=uEnv.txt\0" \
 	"load_bootenv=" \
-		"fatload mmc ${boot_part} ${load_addr} ${bootenv_filename}\0" \
+		"for prefix in / /boot/; do if test -e mmc ${boot_part} ${bootenv_filename}; then load mmc ${boot_part} ${load_addr} ${prefix}${bootenv_filename}; fi; done\0" \
```
Comment (ayufan) [inline include/configs/sun50iw1p1.h]: Should we start from longest prefix first? `/boot`, `/`.

**Review by ayufan — CHANGES_REQUESTED** [review]

File: include/configs/sun50iw1p1.h (hunk 1 above)
Comment (shadeslayer) [reply include/configs/sun50iw1p1.h]: I think having a hierarchy of / and then /boot makes more sense, but I don't really have a strong opinion on this.

Comment (ayufan) [reply include/configs/sun50iw1p1.h]: Can you check what is the order in mainline u-boot? we should have the same then.

Comment (shadeslayer) [reply include/configs/sun50iw1p1.h]: It seems to be /uEnv.txt and /boot/uEnv.txt in include/configs/bav335x.h

**Review by ayufan — APPROVED** [review]

**Pull request merged**